changes/issue-5572-do-not-flush-errors-automatically

```diff
@@ -1 +1 @@
-* Don't clear errors from Redis automatically after running `fleetctl debug errors`, but still allow users to clear errors on read via the `-flush` flag.
+* Fleet no longer clears errors from Redis automatically after running `fleetctl debug errors`, but still allows users to clear errors on Read via the `-flush` flag.
```

Comment:
```suggestion
* Fleet no longer clears errors from Redis automatically after running `fleetctl debug errors`, but still allows users to clear errors on Redis via the `-flush` flag.
```

Reply:
I wasn't sure if it should be "read" or "Redis."

Reply:
not sure why I used `R`, but I meant "read". In any case both work and I like your suggestion!